CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ else ( MSVC )
 endif ( MSVC )
 # End configure editline
 
-IF( WIN32 )
+IF( MSVC )
```

Comment:
#121 replaces some `MSVC` with `WIN32`. Which one is correct / better? @cwyyprog @crypto-ape

Reply:
Depends on context,
we agree on this line.

See [CMake variables](https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/Checking-Platform) for more info.

 - `MSVC` 
    - targets one specific compiler, the Microsoft's Visual C/C++ Compiler
 - `WIN32` 
    - targets whole platform,
    - from my experience this includes `MSVC`, `MINGW` and also cross-compilation on Linux with MINGW compiler